examples/seismic/skew_self_adjoint/example_iso.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from devito import info
+from examples.seismic import RickerSource
```

Comment:
blank line between devito imports and examples imports

Reply:
same comment? fixed